std/stdio.d

```diff
@@ -514,7 +530,10 @@ Throws: `ErrnoException` in case of error.
         }
 
         _p = cast(Impl*) enforce(malloc(Impl.sizeof), "Out of memory");
+        memset(_p, 0x0, Impl.sizeof);
         FILE* handle;
+
+        scope (failure) _p.handle = null;
```

Comment:
Wouldn't this leak memory?

Reply:
Yes, I missed.. thanks for pointing out.